### PR TITLE
show formatted numbers in the paging when over 1,000 pages, fixes #307

### LINKED ISSUE #307: Show formatted numbers in the paging when over 1,000 pages
**Angular version**: 
5.2.11

**ngx-pagination version**:
3.2.1

**Description of issue**:
The paging numbers do not have commas in it when the number of pages go past 1,000.
So the number displays as 4300 instead of 4,300. 
(We use Server-Side Paging in our app so the number of pages can be large)

**Steps to reproduce**:
Set the totalItems to be a large number like 20,000 and make the itemsPerPage to be 5. 

**Expected result**:
Show commas in the button page links for easy reading

**Actual result**:
The pages displayed in the button selection will not have commas in the number.

**Demo**: 
Here is a demo
https://angular-upmtxc.stackblitz.io

Any relevant code:
```

```

### PR DESCRIPTION
**Pull request by mircowidmer**
Have a look at my attempt to fix issue #307.

### CODE REVIEW

**Comment by michaelbromley** [comment]
Hi! Thanks for your contribution. It looks like there are a bunch of tests failing right now - see https://travis-ci.org/michaelbromley/ngx-pagination/builds/550281164?utm_source=github_status&utm_medium=notification

Seems to be caused by invalid template syntax - you can't use a `<span>` in the `else` clause of ngIf.

File: src/template.ts [hunk 1]
```diff
@@ -32,11 +32,13 @@ export const DEFAULT_TEMPLATE = `
             *ngFor="let page of p.pages">
             <a tabindex="0" (keyup.enter)="p.setCurrent(page.value)" (click)="p.setCurrent(page.value)" *ngIf="p.getCurrent() !== page.value">
                 <span class="show-for-sr">{{ screenReaderPageLabel }} </span>
-                <span>{{ page.label }}</span>
+                <span *ngIf="page.label && page.label.constructor.name === 'Number'; else notANumber">{{ page.label | number }}</span>
+                <span #notANumber>{{ page.label }}</span>
```
Comment (michaelbromley) [inline src/template.ts]: This part is not valid Angular template syntax. It should be `<ng-template>`

**Comment by mircowidmer** [comment]
Yes, I just realized it and will fix it!

**Comment by michaelbromley** [comment]
Also just as a heads up, I had another person start working on this issue a few hours ago, but they have not submitted their pull request yet. Of course I'll only be able to merge one of the pull requests, so I can't guarantee your hard work will be merged, sorry!

File: src/template.ts (hunk 1 above)
Comment (mircowidmer) [reply src/template.ts]: It should be fixed with the latest commit.

**Comment by mircowidmer** [comment]
No worries at all, choose the solution that you like the best ;)

File: src/template.ts (hunk 1 above)
Comment (michaelbromley) [reply src/template.ts]: Yep. looks good, tests passing!

**Comment by michaelbromley** [comment]
FYI I merged the other PR for this issue. The reason being that it included tests. Nevertheless thank you for taking the time to contribute.

**Pull request closed without merging**